src/commands/ExecCommand.js

```diff
@@ -27,6 +27,14 @@ export const builder = {
     type: "boolean",
     default: undefined,
   },
+  // This option controls prefix for stream output so that it can be disabled to be friendly
+  // to tools like Visual Studio Code to highlight the raw results
+  prefix: {
+    group: "Command Options:",
+    describe: "Enable prefix for stream output",
+    type: "boolean",
+    default: true,
```

Comment:
So the weird thing about our "durable" options and yargs is that we need to specify `default: undefined` here, otherwise yargs overwrites our lerna.json-based config. Sorry for the weirdness.